doc/site/core/IO.markdown

```diff
@@ -0,0 +1,34 @@
+^title IO Class
+^category core
+
+The IO class can be used to read and write to and from the console.
+
+## Static Methods
+
+### IO.print(obj[, obj2, obj3...])
+Prints any number of things to the console and then prints a new line character. If you don't pass it a string, it will be converted to a string for you. When passed multiple things, Wren outputs one after another.
```

Comment:
Two style nits, but can you add a blank line after the header and hard wrap this paragraph to fit in 80 columns? (Hard wrapping makes diffs and code reviews easier to read.)